native/cocos/math/Quaternion.cpp

```diff
@@ -370,46 +370,76 @@ void Quaternion::toEuler(const Quaternion &q, bool outerZ, Vec3 *out) {
     out->z = attitude;
 }
 
-void Quaternion::fromMat3(const Mat3 &m, Quaternion *out) {
-    CC_ASSERT(out);
+void Quaternion::fromMat3(const Mat3& m, Quaternion* out) {
```

Comment:
Please don't change it. Just do it as other functions in this file.

Reply:
ok